onnxruntime/core/providers/openvino/ov_versions/data_ops.cc

```diff
@@ -95,8 +95,10 @@ std::vector<SupportedOp> supported_op_mode = {
     {"Atan", V_2020_4, {"CPU", "GPU"}},
     {"Atanh", V_2020_4, {"CPU"}},
     {"Atanh", V_2022_1, {"GPU"}},
+    {"Attention", V_2024_5, {"CPU","GPU","NPU"}},
```

Comment:
@ankitm3k 
Currently, we do not have specific operator mapping for the NPU. Instead, we rely on CPU operator support. If model compilation fails on the NPU, it automatically falls back to the CPU. As a result, graph partitioning is determined based on the operator support available on the CPU. So NPU can be removed from all the ops.

You can see this mapping in the constructor of get capability.
[https://github.com/intel/onnxruntime/blob/0981bbf4ca4af4d7216299f15de784f19ce6123a/onnxruntime/core/providers/openvino/ov_versions/capability.cc#L34]